### PR TITLE
todo-create

### PR DESCRIPTION
**Pull request by darmhoo**
#### What does this PR do?
- (replace me) Explain WHY this PR is required. E.g.: What feature it adds, what problem it solves, …

#### Description of Task proposed in this pull request?
- (replace me) A description of WHAT changed in the codebase. Additionally add the reasoning if it's complex or abstract. 

#### How should this be manually tested (Quality Assurance)?
- Explain how to manually test this feature. For example if changes were made in
the UI or in the API, explain where and if any specific access is needed.

#### What are the relevant pivotal tracker stories?
- [Story ID:  replace text with story id](paste Pivotal Tracker link here)

#### Any background context you want to add (Operations Impact)?
- (replace me) List the steps that must be taken for this PR to work.
E.g.: `npm install`, `Add "github_key" to environment variables, ...`

Be sure to include all systems that needs to be changed or which system is
affected by the change (Ex: Requires Styled Components to be installed and
configured in `babel.rc`, file to be updated).

#### What I have learned working on this feature: [If you don't put anything here, you are doing it wrong!]
- (replace me) I learned how to create a pull request template

#### Screenshots: [If you made some visual changes to the application please upload screenshots here, or remove this section]

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: database/migrations/new/2021_03_16_135751_create_feed_backs_table.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('feed_backs', function (Blueprint $table) {
+            $table->dropForeign('feed_backs_todo_id_foreign');
```
Comment (teebarg) [inline database/migrations/new/2021_03_16_135751_create_feed_backs_table.php:39]: why do we have this???

File: database/migrations/new/2021_03_16_095556_create_todos_table.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('todos', function (Blueprint $table) {
+            $table->dropForeign('todos_customer_id_foreign');
```
Comment (teebarg) [inline database/migrations/new/2021_03_16_095556_create_todos_table.php:39]: why are we dropping foreign key first???

Comment (darmhoo) [reply database/migrations/new/2021_03_16_095556_create_todos_table.php:39]: I encounterd foreign key error when rolling back. I had to drop foreign key first

File: database/migrations/new/2021_03_16_135751_create_feed_backs_table.php (hunk 1 above)
Comment (darmhoo) [reply database/migrations/new/2021_03_16_135751_create_feed_backs_table.php:39]: The same here

**Review by teebarg — APPROVED** [review]

**Pull request merged**